book/src/asm/functions.md

```diff
@@ -0,0 +1,74 @@
+# Functions
+
+Machine functions are the entry points to this machine. They can be called from another machine or from the outside.
+
+For static machines, functions simply indicate which columns should be exposed as inputs and outputs. They do not contain statements, since the business logic of all functions in the static machine is defined directly in constraints. We refer to the previous example [here](./machines.md#static-machines).
```

Comment:
```suggestion
For static machines, functions simply indicate which columns should be exposed as inputs and outputs. They do not contain statements, since the business logic of all functions in a static machine is defined directly in constraints. We refer to the previous example [here](./machines.md#static-machines).
```